tests/test_coastal_vulnerability.py

```diff
@@ -713,26 +713,67 @@ def test_complete_run(self):
         args['slr_field'] = 'Trend'
         coastal_vulnerability.execute(args)
 
-        actual_values_df = pandas.read_csv(
-            os.path.join(args['workspace_dir'], 'coastal_exposure.csv'))
-        expected_values_df = pandas.read_csv(
-            os.path.join(REGRESSION_DATA, 'expected_coastal_exposure.csv'))
-        pandas.testing.assert_frame_equal(
-            actual_values_df, expected_values_df, check_dtype=False)
-
-        # Also expect matching shore_id field in all tabular outputs:
-        intermediate_csv = pandas.read_csv(
-            os.path.join(
-                args['workspace_dir'],
-                'intermediate/intermediate_exposure.csv'))
-        habitat_csv = pandas.read_csv(
-            os.path.join(
-                args['workspace_dir'],
-                'intermediate/habitats/habitat_protection.csv'))
-        pandas.testing.assert_series_equal(
-            actual_values_df['shore_id'], intermediate_csv['shore_id'])
-        pandas.testing.assert_series_equal(
-            actual_values_df['shore_id'], habitat_csv['shore_id'])

```

Comment:
I think this block is still important to test. We need to assert that each table has the `shore_id` field and that it can be used to correctly join the tables. It looks like these assertions are comparing the tables within the current run - not comparing against regression data - so I think this check is still useful and resilient to a changing order across model versions.

Reply:
Yep, I concur.  CV's collection of tables form a nice relational database associated by `shore_id`, so while the FID might not be consistent, the `shore_id`s should allow for the same records to be joined across multiple tables.

Reply:
Added this back in